### ISSUE TITLE
Consider changing library function call syntax we use

### ISSUE THREAD

**Issue by frangio** [OpenZeppelin/openzeppelin-contracts#2484]
We want to consider using the "fully qualified" library function call syntax, such as `SafeERC20.safeTransfer(_token, ...)` as opposed to `_token.safeTransfer`, because it can be confusing where the function is actually defined.

Arguments by @Amxx in #2480:

>Because it avoid any ambiguity over the fact that this is a library being called, and not a builtin function. Imagine if `Address` contained a `transfer` function (good thing it doesn't) ... then what would address.transfer be ? The library, the built-in function ?
>
>I think the `using for` syntax is great for safemath. I also think it is acceptable when the first type is clearly not a contract (EnumerableMap/EnumerableSet) ... For addresses I'm not sure ... for contracts I think it's a terrible idea.
>
>In `TokenTimelock` there is an IERC20 token that is called using `_token.safeTransfer(beneficiary(), amount);`. I think there is a real ambiguity between a potential `safeTransfer` function included in IERC20, and a safeTransfer library function that is used for IERC20 (which is the case here).
>
>TLDR: 
>In the particular case, I wouldn't oppose doing `_beneficiary.sendValue(address(this).balance);`, but there are cases where I think this syntax is to be avoided.

**Comment by frangio** [comment]
We agree to use `using for` syntax for types like strings, uints, and adresses, but not for contract types since the function call can be confused with a method call.
